ui/core/src/hooks/addLiquidityCalculator.ts

```diff
@@ -105,11 +105,11 @@ export function usePoolCalculator(input: {
     const bAmount = toField.fieldAmount.value;
     if (!aAmount || !bAmount) return "";
     if (bAmount.equalTo("0")) return "";
-    return `${aAmount
+    return `<span class="number">${aAmount
```

Comment:
I don't think this markup should be here. We would be much better off returning primitives from here and formatting them in the consumer.

Reply:
Agree. I'll need this as two separate strings for styling purposes. Is it fine if I returned an object here with `value` and `copy`?

Reply:
yup it should return whatever is the easiest for the UI without knowing anything about the way the UI is rendering the info